sbin/spark-daemon.sh

```diff
@@ -169,6 +169,7 @@ case $option in
       if [[ $(ps -p "$TARGET_ID" -o args=) =~ $command ]]; then
         echo "stopping $command"
         kill "$TARGET_ID"
```

Comment:
Concretely, can there be an `if` statement here to check whether `kill` succeeded, and only `rm` if so?